llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1677,6 +1677,15 @@ class VPWidenIntrinsicRecipe : public VPRecipeWithIRFlags {
         MayWriteToMemory(CI.mayWriteToMemory()),
         MayHaveSideEffects(CI.mayHaveSideEffects()) {}
 
+  // VPWidenIntrinsicRecipe(CastInst &CI, Intrinsic::ID VectorIntrinsicID,
+  //                        ArrayRef<VPValue *> CallArguments, Type *Ty,
+  //                        DebugLoc DL = {})
+  //     : VPRecipeWithIRFlags(VPDef::VPWidenIntrinsicSC, CallArguments, CI),
+  //       VectorIntrinsicID(VectorIntrinsicID), ResultTy(Ty),
+  //       MayReadFromMemory(CI.mayReadFromMemory()),
+  //       MayWriteToMemory(CI.mayWriteToMemory()),
+  //       MayHaveSideEffects(CI.mayHaveSideEffects()) {}
+
```

Comment:
This should drop?

Reply:
It seems...still there?

Reply:
I would say it's not right way to just put that in the comment...:P

Reply:
OK, thanks :)